app/assets/javascripts/html5_webshim.js

```diff
@@ -7,7 +7,14 @@ $(document).on('turbolinks:load', function(){
       type: 'date',
       date: {
         startView: 2,
-        openOnFocus: true
+        openOnFocus: true,
+        popover: {
```

Comment:
i love popovers.  My mom used to make them whenever she served standing rib roast.  So yummy!